tests/integration/api_server/test_dependencies.py

```diff
@@ -0,0 +1,64 @@
+import contextlib
+import json
+import time
+
+import docker
+import requests
+
+from .dependency_verification_service import gen_test_bundle
+
+
+@contextlib.contextmanager
+def docker_container(port=5000):
+    client = docker.from_env()
+    # TODO: make test_dest dir according to travis fs
+    test_dir = '../bento-tests/int_test'
```

Comment:
use pytest's tempdir fixture?

Reply:
take a look at the example in `tests/integration/api_server/conftest.py`

Reply:
pytest has session level fixture and autouse flag to do things like contextmanager. It allows different test functions share the same host.